docs/developer/explanations/decisions/0004-repository-structure.rst

```diff
@@ -55,24 +55,40 @@ During this process, the folder structure should incrementally be changed to
     │       │   ├── __init__.py
     │       │   ├── backends
     │       │   │   ├── __init__.py
-    │       │   │   ├── _aioca.py
-    │       │   │   └── _p4p.py
+    │       │   │   ├── signal_backend.py
+    │       │   │   └── sim.py
     │       │   ├── devices
-    │       │   ├── signals
+    │       │   │   ├── __init__.py
+    │       │   │   ├── device_collector.py
+    │       │   │   └── ...
     │       │   ├── epicsdemo
```

Comment:
Please can we move `ophyd_async.core.epicsdemo` to `ophyd_async.epics.demo`

Reply:
yep, on it